components/raftstore/src/store/fsm/peer.rs

```diff
@@ -1943,11 +1906,94 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
         )
     }
 
+    /// Check if merge target region in kv engine is fresher than target region.
+    /// It should be called when target region is not in region map in memory.
+    /// If everything is ok, the answer should always be true because PD should ensure all target peers exist.
+    /// So if not, error log will be printed and return false.
+    fn is_local_merge_target_region_fresher(&self, target_region: &metapb::Region) -> Result<bool> {
```

Comment:
How about `is_merge_target_outdated`?

Reply:
I think the original name is fine 🤣 because I want to emphasize `local_target_region`

Reply:
The original name is too long, we usually use three to four words for naming. And using `fresher` here is very weird. A more up to date local target region means `target_region` to be checked is outdated.